mobile-app/app/components/TransactionCard.tsx

```diff
@@ -0,0 +1,120 @@
+import { ThemedViewV2, ThemedTextV2, ThemedTouchableOpacityV2 } from '@components/themed'
+import BigNumber from 'bignumber.js'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { StyleProp, ViewStyle } from 'react-native'
+
+interface TransactionCardProps {
+  maxValue: BigNumber
+  onChange: (amount: string, type: AmountButtonTypes) => void
+  status?: string
+  containerStyle: StyleProp<ViewStyle>
+}
+
+export enum AmountButtonTypes {
+  twentyFive = '25%',
+  half = '50%',
+  seventyFive = '75%',
+  max = 'MAX'
+}
+
+export function TransactionCard ({ maxValue, onChange, status, containerStyle, children }: React.PropsWithChildren<TransactionCardProps>): JSX.Element {
+  return (
+    <ThemedViewV2
+      light={tailwind('bg-mono-light-v2-00', {
+        'border-0.5 border-mono-light-v2-800': status === 'active',
+        'border-0.5 border-red-v2': status === 'error'
```

Comment:
we can remove theme styling for `error` since they are added in `style`

Reply:
done, removed `error` theme styling